cmd/virt-handler/virt-handler.go

```diff
@@ -68,19 +71,41 @@ func main() {
 		panic(err)
 	}
 
+	// Create event recorder
+	coreClient, err := kubecli.Get()
+	if err != nil {
+		panic(err)
+	}
+	broadcaster := record.NewBroadcaster()
+	broadcaster.StartRecordingToSink(&kubecorev1.EventSinkImpl{Interface: coreClient.Events(api.NamespaceDefault)})
+	recorder := broadcaster.NewRecorder(kubev1.EventSource{Component: "virt-handler", Host: *host})
```

Comment:
Looks like we start to expose the component name `virt-handler` here to the cluster.
Reminds me that we should look if handler is a good suffix for such acomponent.